.github/workflows/test-multiple-versions.yml

```diff
@@ -33,9 +33,9 @@ jobs:
           - 18.1.0
           - 18.2.0
           - 18.3.1
-          - 19.0.0-rc.0
-          - 19.0.0-rc-5c56b873-20241107
-          - 0.0.0-experimental-5c56b873-20241107
+          - 19.0.0-rc.1
+          - 19.0.0-rc-e1ef8c95-20241115
+          - 0.0.0-experimental-e1ef8c95-20241115
```

Comment:
Here's the list of commits for the change:

https://github.com/facebook/react/compare/5c56b873...e1ef8c95

Reply:
More precisely, it's between these commit that cause the error:
https://github.com/facebook/react/compare/380f5d67...b01722d5

Reply:
So, `enableSiblingPrerendering` should only be the meaningful change.